styles.css

```diff
@@ -0,0 +1,4 @@
+h1 {
```

Comment:
[ ] kindly use a CSS class for your header. it's one of the HTML and CSS best practices and it's on the project requirements. 👇
![css](https://user-images.githubusercontent.com/62156607/204653368-8558cbf9-7470-4d85-a7c4-7801eef8c0cc.PNG)

Reply:
Thank you for your guidance @nedjwab 🥇 

I follow your instructions and change the pull request title and descripction. Also I add the .gitignore file to the repository.

I await any further correction.

Reply:
I already did a new commit with the style correction adding the class to the h1 tag.

Reply:
Hello @nedjwab 
I would like to know if you check the changes in my project. 😰